docs/data-guidelines.md

```diff
@@ -380,3 +380,50 @@ For example, `HTMLMediaElement`'s `play()` method returns a promise, recorded li
 ```
 
 This guideline is based on discussion in [#11630](https://github.com/mdn/browser-compat-data/pull/11630).
+
+## Global APIs
+
+An API is considered global when it is available in at least two global scopes. Such APIs are recorded in the `api/_globals/` folder.
+
+These are the web's global scopes at the time of writing:
+
+- [`Window`](https://developer.mozilla.org/en-US/docs/Web/API/Window)
+- [Workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API): [`DedicatedWorkerGlobalScope`](https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope), [`SharedWorkerGlobalScope`](https://developer.mozilla.org/en-US/docs/Web/API/SharedWorkerGlobalScope), [`ServiceWorkerGlobalScope`](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope) (these inherit from the abstract [`WorkerGlobalScope`](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope))
+- [Worklets](https://developer.mozilla.org/en-US/docs/Web/API/Worklet): `AnimationWorkletGlobalScope`, [`AudioWorkletGlobalScope`](https://developer.mozilla.org/en-US/docs/Web/API/AudioWorkletGlobalScope), `LayoutWorkletGlobalScope`, `PaintWorkletGlobalScope` (these inherit from the abstract `WorkletGlobalScope`)
+
+For example, the [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/fetch) method is global because it is available in `Window` and `WorkerGlobalScope`, recorded like this in `api/_globals/fetch.json`:
+
+```json
+{
+  "api": {
+    "fetch": {
+      "__compat": {},
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "support": {}
+        }
+      }
+    }
+  }
+}
+```
+
+Note that all APIs defined on the [`WindowOrWorkerGlobalScope`](https://html.spec.whatwg.org/multipage/webappapis.html#windoworworkerglobalscope-mixin) mixin are considered global.
+
+These APIs are also considered global although they're not available in _all_ workers:
+
+- The `cookieStore` property, available in `Window` and `ServiceWorkerGlobalScope`.
+- The `requestAnimationFrame()` function, available in `Window` and `DedicatedWorkerGlobalScope`.
+
+TODO: should we really still use "worker_support" here, which makes it look like these are available in all workers?
+
+However, these APIs are _not_ considered global:
+
+- `close()` methods are available in `Window`, `DedicatedWorkerGlobalScope` and `SharedWorkerGlobalScope` but with different meanings. For `Window` it closes the user-visible window and in workers it stops the worker. This isn't considered one global function despite sharing the name and being available in three global scopes.
+
+TODO: This exception doesn't follow from the rule as written. Some tweak is needed.
```

Comment:
The tweak I have in mind is to say it has to be available in `Window` and one other global scope. I don't want the guideline to require an evaluation of sameness, since that can't be encoded as a rule in BCD tooling like mdn-bcd-collector.

Reply:
But IDL does encode such sameness (by using mixins), maybe some tweak can be done there.

Reply:
Generally yes, but specs don't *have* to use `WindowOrWorkerGlobalScope` for things that are the same. `cookieStore` is an example where the sameness can't really be seen from IDL:
https://wicg.github.io/cookie-store/#Window
https://wicg.github.io/cookie-store/#ServiceWorkerGlobalScope

Reply:
@foolip It sort of sounds to me that you could drop the "However, these APIs are _not_ considered global" section and wrap this up. Does that sound right?

Reply:
I've simplified the guideline now, throwing out all of the TODOs and complexity.